### DISCUSSION TITLE
When debugging with ssh Engine, debug variables cannot be displayed in the variables bar

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1323] [Q&A] [answered]
Environment：win10，pyscripter5.0.1
Remote debugging starts, but variables bar does not have any display, won't remote debugging variables be displayed in variables bar？

![image](https://github.com/user-attachments/assets/c86e5071-e73f-4934-b4ef-762227f0a257)

![image](https://github.com/user-attachments/assets/def2ddab-7fc9-4b86-8d7a-0de53109176b)

**Comment by pyscripter** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/76fbcd10-7c4a-4d08-9871-6f59a4b0a8fa)

The option exists because it can slow things down.

**Reply by wilikeit** [reply]
Thank you. That solves my problem. Why doesn't this switch be turned on by default, Is there any consideration? This setting does not seem to be known to novices, who routinely assume that remote debugging automatically displays variables.

**Reply by pyscripter** [reply]
As explained above, enabling this option can slow things down with a slow ssh connection.
